libdirections-onboard/src/main/java/com/mapbox/navigation/route/onboard/model/Endpoint.kt

```diff
@@ -0,0 +1,18 @@
+package com.mapbox.navigation.route.onboard.model
+
+import com.mapbox.navigation.navigator.model.EndpointConfig
+
+data class Endpoint(
```

Comment:
Same as above, just `TileEndpointConfiguration` is enough. The onboard router always depends on the navigator so can use its types.